.github/workflows/merge-conflict-check.yml

```
@@ -0,0 +1,65 @@
+name: Merge Conflict Check Workflow
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)